modules/dxp/apps/saml/saml-opensaml-integration/src/main/java/com/liferay/saml/opensaml/integration/internal/field/expression/handler/MembershipsUserFieldExpressionHandler.java

```diff
@@ -91,11 +91,13 @@ public void bindProcessorContext(
 									"Unable to create User Group",
 									portalException);
 							}
-
-							continue;
 						}
 					}
 
+					if (userGroup == null) {
```

Comment:
Should we do this instead @rafaprax 

```
if (userGroup != null) {
   userGroupIds.add(userGroup.getUserGroupId());
}
```

Just a thought

Reply:
I believe BChan could apply the "Buy Space" rule in this case.